client/src/components/screens/VariantDetails/index.jsx

```diff
@@ -548,37 +476,202 @@ class VariantDetailsScreen extends React.Component {
     actions.fetchVariantDetails(uid);
   }
 
-  getConsequences() {
+  getConsequences(gene) {
     const {
       variantDetails,
     } = this.props;
 
+    const {
+      openGeneTable,
+      isVepOpen,
+    } = this.state;
+
     if (variantDetails) {
       const { data } = variantDetails;
       if (data) {
-        const { consequences } = data;
-        return consequences;
+        let geneToShow = gene;
+        if (!openGeneTable.includes(gene[0].symbol)) {
+          const canonicalLine = filter(gene, { canonical: true });
+          geneToShow = canonicalLine.length === 0 ? [gene[0]] : canonicalLine;
+        }
+
+        return geneToShow.map((g, index) => {
+          const getVepTag = () => {
+            switch (g.impact) {
+              case 'HIGH':
+                return (
+                  <Tag className="consequenceTable__tag" color="red"> HIGH </Tag>
```

Comment:
À traduire ou ça reste toujours en anglais?

Reply:
j'allais toute les faire à la fin

Reply:
Si c'est un part 1 of X c'est bon alors